.github/ISSUE_TEMPLATE/ixd_user_story.md

```diff
@@ -0,0 +1,35 @@
+---
+name: 🧑‍🎨 IxD User story
```

Comment:
I'm not sure if we need to document this somewhere, but I'm still a bit fuzzy on who/when this item gets created. My guess is that this is the workflow:

- Someone creates an issue for a new component
- The implementer of the new component creates an IxD issue with details around the requirements
- The interaction designer creates the issue for the visual design once the interaction design is complete

Some questions:
- Is that general workflow correct?
- Who is ultimately defining all the requirements of a component and getting approval for those? Some PO? The implementer? The interaction designer?
- Does this mean we need to plan components farther in advance than we do now? For example, the current workflow is Design -> Implementation Spec -> Implementation, but it sounds like maybe it now is Requirements Gathering (done by implementer??) -> IxD Design -> Visual Design -> Implementation Spec -> Implementation.

Reply:
> Wait, so is the expectation that I create a ViD issue?

No, but let me update the diagram to make that clear – 
![image](https://user-images.githubusercontent.com/1458528/231889474-36a5781a-23df-4f71-8a97-bd939bf7e6aa.png)

I'll create both IxD and Visual Design work items, and we'll keep them separate; using the new component request issue/work item to capture any shared information/context. So, if the IxD requirements are minimal, that's completely fine. This way we can delay the creation of the visual design work items until it's clear that we need them, and the requirements are clear.